### DISCUSSION TITLE
Projektthema

### DISCUSSION THREAD

**Discussion by codajoda** [SE-TINF22B2/G3-ApeRepublic#2] [Ideas]
Hier wird über das Projektthema diskutiert.
Allgemein soll die Anwendung gut erweiterbar sein (Stichwort Microservices), da wir sogar über die Praxisphase hinaus Zeit haben, wodurch es schwierig wird, den Arbeitsaufwand abzuschätzen. Also eine Anwendung mit vielen kleinen unabhängigen Funktionen wäre optimal. Wie und mit welchen Architekturen die Anwendung realisiert werden soll, wird entsprechend in einer anderen Discussion besprochen.

Vorgaben: Die Anwendung enthält

Frontend
Backend
Api
Datenbank
Zeit: ~5 Monate

**Comment by codajoda** [comment]
News-Plattform mit personalisierter Ansicht aller Infos (News-Seiten, Aktien, ...). Es sollen auch Mini-Games möglich sein.
Banking App -> API Call für Aktien und/oder EZB Leitzins und/oder Währungen
Neobroker -> API Call für gefühlt alles lol
MonkeyType -> API Call für Texte
Cookie Clicker
Mensa App -> die Studenten können per App bestellen, zahlen und dann nur einfach bei der Kasse abholen. (Nicht nötig mehr in einer langen Schlange warten)
Doulingo App für Informatik ---> https://www.gutefrage.net/frage/gibt-es-sowas-wie-duolingo-fuer-informatik
Neues Platform aus der man Artikel von verschiedenen Medien zu einem Theman erhält, um Meinungsvielfalt abzubinden, inklusive Diskussionsforum
MeowSurvey -> Umfrage App, Bei Teilnahme an Umfrage wird man belohnt mit Katzenbild (CatPictureAPI)

**Comment by Marc738** [comment]
# Diskussion abgeschlossen
Ergebnis: Neobroker mit API Calls für Wertpapierperformance
